chain/cosmos/cosmos_chain.go

```diff
@@ -396,6 +396,14 @@ func (c *CosmosChain) UpgradeProposal(ctx context.Context, keyName string, prop
 	return c.txProposal(txHash)
 }
 
```

Comment:
Can you add comment specifying this is for chains that are using thee `v1beta1` version of gov module.